### PR TITLE
py-bidscoin, py-multiecho: add new packages

### PR DESCRIPTION
**Pull request by manuelakuhn**
https://github.com/Donders-Institute/bidscoin/tree/3.7.4
https://github.com/Donders-Institute/multiecho/tree/0.28

### CODE REVIEW

**Comment by manuelakuhn** [comment]
ping @adamjstewart

**Review by tldahlgren — COMMENTED** [review]
Confirmed sha256 for both packages

File: var/spack/repos/builtin/packages/py-multiecho/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class PyMultiecho(PythonPackage):
+    """Combine multi-echoes from a multi-echo fMRI acquisition."""
+
+    homepage = "https://github.com/Donders-Institute/multiecho"
+    pypi = "multiecho/multiecho-0.28.tar.gz"
+
+    version("0.28", sha256="d0459bd03398547116d8e989b2d2b7922af0ae7ae77e233794dd7253a2abced3")
+
+    depends_on("python@3.6:3.9", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiecho/package.py:17]: I don't see any `python_requires` so the supported versions of Python should be unrestricted, I would remove this line.

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-multiecho/package.py:17]: The python restriction I got from here:
https://github.com/Donders-Institute/multiecho/blob/0.28/pyproject.toml#L11

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-multiecho/package.py:17]: Hmm, I'm not actually sure if setuptools reads that part of the file, but let's keep it for now.

**Review by adamjstewart — APPROVED** [review]

**Comment by manuelakuhn** [comment]
@spackbot run pipeline

**Pull request merged**